src/h5web/vis-packs/core/utils.ts

```diff
@@ -228,3 +236,11 @@ export function isScaleType(val: unknown): val is ScaleType {
     typeof val === 'string' && Object.values<string>(ScaleType).includes(val)
   );
 }
+
+function isSupported(val: number) {
+  if (val > 0) {
+    return val > MIN_POSITIVE_VALUE && val < Number.MAX_VALUE;
+  }
+
+  return val > -Number.MAX_VALUE && val < Number.MAX_VALUE;
```

Comment:
`&& val < -Number.MIN_POSITIVE_VALUE` ? But maybe you can simplify this with `Math.abs`.